### DISCUSSION TITLE
手动连接桩连线，移动节点可以自适应连接桩

### DISCUSSION THREAD

**Discussion by kaix-w** [antvis/X6#2383] [Q&A]
节点通过连接桩（矩形节点，四周都有连接桩）连线，路由是manhattan路由，希望移动节点的时候，连线可以自动找最近的连接桩位置进行连接。
看了issue里面的一个方法，是修改连接点以及锚点，[如何让线与节点的链接点事指定锚点，并且自适应？](https://github.com/antvis/X6/issues/749)，在 midSide node-anchor 中增加配置:direction:'H | 'V'，但是测试发现并无效果，
如果是通过addEdge方法，指定source和target的话（不指定连接桩，但是节点设置的有），移动节点是可以自适应的，
但是如何让手动通过连接桩连线的时候实现这个效果

**Comment by mtiger95** [comment]
有同样的需求，考虑过在移动节点过程中不断去获取各节点坐标进行对比，然后计算出最近的进行连接。后来产品砍了这个需求。
